tests/unit/mesh/data/test_gridfromoptions.cxx

```diff
@@ -419,6 +426,7 @@ TEST_F(GridFromOptionsTest, CoordinatesXlowRead) {
 }
 
 TEST_F(GridFromOptionsTest, CoordinatesYlowInterp) {
+#ifndef COORDINATES_USE_3D
```

Comment:
What's the issue with this test?

If it really can't be done, might be nicer to put the `#ifndef` outside the test so that it's completely skipped